lib/pre-without-class.js

```diff
@@ -15,7 +15,8 @@ exports.preWithoutClass = {
   name: "pre_without_class",
   desc: "pre_without_class_desc",
   check: function checkPreWithoutClass(rootElement) {
-    let presWithoutClass = rootElement.querySelectorAll("pre:-moz-any(:not([class]), [class=''])");
+    //let presWithoutClass = rootElement.querySelectorAll("pre:-moz-any(:not([class]), [class=''])");
+    let presWithoutClass = rootElement.querySelectorAll(":not([class]), [class='']");
```

Comment:
Doesn't this match all elements without class? It should only match `<pre>` elements without class.